scenarios/anomaly-detection/build.gradle

```diff
@@ -97,23 +91,31 @@ shadowJar {
     }
 }
 
+task scriptAnomalyDetection(type: CreateStartScripts) {
+    outputDir = file('build/scripts')
+    mainClassName = 'io.pravega.anomalydetection.ApplicationMain'
+    applicationName = 'anomaly-detection'
+    defaultJvmOpts = ["-Dlog4j.configuration=file:conf/logback.xml"]
+    classpath = files(jar.archivePath) + sourceSets.main.runtimeClasspath
+}
+
 distributions {
     main {
         baseName = archivesBaseName
         contents {
+            into('bin') {
+                from project.scriptAnomalyDetection
+            }
             into('lib') {
                 from shadowJar
                 from(project.configurations.shadow)
+                from(jar)
+                from(project.configurations.runtime)
```

Comment:
Seems we are trying to allow the application to run standalone, i.e. without `flink run ...`.   As such, we're copying the `runtime` libs that normally would be provided by Flink.  Is this because the anomaly app has a standalone mode of use or does the app always use Flink?

Reply:
Yes, according to the documentation, the application should allow to be executed via a script `bin/anomaly-detection` and submitting the `jar` to Flink (currently, it works in both scenarios).